tg_bot/modules/sql/users_sql.py

```diff
@@ -138,6 +139,35 @@ def get_user_num_chats(user_id):
         SESSION.close()
 
 
+def get_chatname_by_chatid (chat_id):
+    try:
+        return SESSION.query(Chats).get(str(chat_id)).chat_name
```

Comment:
what if that chat doesnt exist, you'll get a None from the db, and youll get a `nonetype has no attr chat_name` error